store/modules/events.ts

```diff
@@ -1,3 +1,4 @@
+type CommunityEventsToShowByDate = 'past'|'upcoming'
```

Comment:
If changing the name, the name of the type can be simplified to `CommunityEventSet` or alike.